tools/nvm/nvm-guide.md

```diff
@@ -0,0 +1,24 @@
+# Introduction
+
+### What is nvm?
+
+**nvm** stands for **Node Version Manager** and is a tool that allows developers to install and manage multiple versions of **Node.js** platform.
+
+# Commands
+
+### Check versions installed
+
+```sh
+nvm ls
+```
+### Install a new version
+
+```sh
+nvm install <node-version>

```

Comment:
As far as it concerns the installation of a node version using nvm, the command according to nvm README is [nvm install \<node-version\>](https://github.com/creationix/nvm#usage).

There is a command [nvm ls-remote](https://github.com/creationix/nvm#listing-versions) that provides the versions available for installment.

In [How To Install Node.js on Ubuntu 16.04](https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-16-04) it uses `nvm ls-remote` to check the available versions and then `npm install <node-version>` to install a specific version.